### DISCUSSION TITLE
facing an issue with uploading files

### DISCUSSION THREAD

**Discussion by vsk0718** [community/community#45696] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

pls help me out for uploading files. it is showing an error again and again

**Comment by iamsuiux** [comment]
Maybe you can reference the below links.
https://stackoverflow.com/questions/5587973/javascript-upload-file
https://www.npmjs.com/package/upload-js
https://github.com/Buzut/huge-uploader
https://www.geeksforgeeks.org/file-uploading-in-node-js/

**Comment by surjendu104** [comment]
What is the error message??

**Comment by vsk0718** [comment]
issue automatic resolved ... thank u!

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @vsk0718 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
